### PR TITLE
Remaster gui

### PR DESCRIPTION
**Pull request by bridCquinn**
Changed the home Page:
 Added the advanced settings area
 Moved elements around 
The text colors were changed for ease of reading
Added Margin for Blocks

### CODE REVIEW

**Comment by szczura** [comment]
This does not accomplish the acceptance criteria for the task assigned:

- Windows and elements need to be scaled such that the entire window is taken up regardless of resolution
- elements of UI need to have min/max scaling such that they are still readable at reasonable resolutions and aspect ratios.

image
image

**Comment by szczura** [comment]
Can you add screenshots when you make updates?

File: WPF/SeeShells/SeeShells/UI/Node/InfoBlock.cs [hunk 1]
```diff
@@ -67,7 +67,7 @@ public void ToggleInfo()
             }
             else
             {
-                this.Width = 450;
+                this.Width = Double.NaN;
```
Comment (szczura) [inline WPF/SeeShells/SeeShells/UI/Node/InfoBlock.cs]: This isn't going to work as the solution to the info-boxes being cut off. 

A user can't see all the information:
image

**Comment by bridCquinn** [comment]
Beginning Page
This is when it first opens

everything open
This is with everything open

everything open
This is the vertical view

**Comment by klaki892** [comment]
> Beginning Page
> This is when it first opens

I like this design, but all of this empty space with the word "Offline" Now just seems...wrong? A random person wouldnt know what we meant by just the word "offline"
 
**@Team** wouldn't this make more sense if we made it a RadioButton with the options "Live Registry" and "Offline Registry" or "Registry from file(s)"?

**Comment by bridCquinn** [comment]
This is the rescaled GUI timeline page.
timlinepage

This is now how the hover is done.
hover fix

What was done:
- The timeline and home page now adapts to the size of the screen which the user may put it in. 
- The size of the window which opens has been modified to give the optimal viewing experience
- The min sizes have been set to eliminate a UI too small to read
- The Home page now has an advanced section as to not overwhelm the new user with configuration
- The hovering has been changed to have a wrapping functionality

The Problem (Has now been overcome):
The issue with the UI never taking up the entire space of the screen was caused by the tool bar. It was wrapping the entire window in a WrapPanel, so I changed it to a dock panel and simply docked it to the top. After this was done, all of the grids gained their normal functionality.

**Comment by bridCquinn** [comment]
radio button reg
This is what the opening page looks like now with the radio buttons (the online registry is the pre-selected choice, to stay with the idea of only having to press the Parse button for an online registry)

radiobutton open
This is what it looks like when everything is open.

@klaki892 Thanks for the suggestion!

**Comment by bridCquinn** [comment]
Resolves #273 . Resolves #257 . Resolves #188 . Resolves #290

File: WPF/SeeShells/SeeShells/UI/Node/InfoBlock.cs (hunk 1 above)
Comment (bridCquinn) [reply WPF/SeeShells/SeeShells/UI/Node/InfoBlock.cs]: This has now been changed in the updates below, pictures are provided.

**Comment by klaki892** [comment]
> Resolves #273 . Resolves #257 . Resolves #188 . Resolves #290

The Resolves keyword only works in the opening message of a PR
You can add these issues with the sidebar saying "linked issues"

**Pull request merged**